### PR TITLE
check formatting only in changed files in pre-commit hook

### PR DESCRIPTION
**Pull request by rasom**
On my computer `git commit ..` takes now ~5-7s when it was 21-24s, still slow but faster.

status: ready

### CODE REVIEW

**Review by janherich — APPROVED** [review]

**Review by cammellos — APPROVED** [review]

File: project.clj [hunk 1]
```diff
@@ -14,9 +14,9 @@
   :plugins [[lein-cljsbuild "1.1.7"]
             [lein-re-frisk "0.5.8"]
             [lein-cljfmt "0.5.7"]
-            [lein-githooks "0.1.0"]]
+            [rasom/lein-githooks "0.1.3"]]
```
Comment (jeluard) [inline project.clj:17]: Could we consider merging those changes upstream?

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/h8ci4i for branch PR-4526

<blockquote><div><strong><a href="https://i.diawi.com/h8ci4i">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/ycB9QW for branch PR-4526

<blockquote><div><strong><a href="https://i.diawi.com/ycB9QW">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

File: project.clj (hunk 1 above)
Comment (rasom) [reply project.clj:17]: @jeluard latest changes in that repo are 3 years old, i don't really want to bother about this tbh

Comment (rasom) [reply project.clj:17]: we can move it to `status-im/lein-githooks` though

**Pull request merged**